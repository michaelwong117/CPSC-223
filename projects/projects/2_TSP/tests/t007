#!/bin/bash
# Farthest with ties

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stdout=stdout.out -stderr=/dev/null ./TSP /c/cs223/hw2/Tests/t007.in -farthest < /dev/null
cat stdout.out  | cut -b1-32
