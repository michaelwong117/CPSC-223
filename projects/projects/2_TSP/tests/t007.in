5
AAA BBB CCC DDD EEE
0.0 0.0
30.0 -180.0 
-30.0 -180.0
0.0 90.0
0.0 -90.0
