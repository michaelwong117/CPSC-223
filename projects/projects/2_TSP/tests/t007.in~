7
AA3 BBB AA1 AA5 CCC AA4 AA2
0.0 0.0
10.0 29.0
11.0 0.0
-11.0 0.0
-10.0 29.0
-9.0 0.0
9.0 0.0
