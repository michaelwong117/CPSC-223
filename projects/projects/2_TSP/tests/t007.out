-farthest         :     56746.30
